### PR TITLE
Update simulation text for section 06

### PR DESCRIPTION
**Pull request by ajlee21**
**Did you add yourself as a [contributor](https://github.com/Benjamin-Lee/deep-rules/blob/master/contributors.md) if this is your first contribution?**

- [x] Yes, I added myself or am already a contributor

**Any more details?**

This PR extends from the discussion in https://github.com/Benjamin-Lee/deep-rules/pull/218.

I have added some additional text about how simulation methods can be used to help you understand your data, in order to help tie this paragraph to the central idea of this section `knowing your data/problem`.

@pstew @cgreene Please let me know if these edits make sense, if I misunderstood something as I know I'm new to contributing to this manuscript.

### CODE REVIEW

File: content/06.know-your-problem.md [hunk 1]
```diff
@@ -32,10 +32,15 @@ For a large dataset, overfitting may not be a concern, but the modeling power of
 It is important to note that molecular or imaging datasets often require appropriate clinical or demographic data to support robust analyses; this must always be balanced with the need to protect patient privacy (see [Tip 10](#privacy)).
 Looking at these annotations can also clarify the study design (for example, by seeing if all the individuals are adolescents or women) or at least help the analyst employing deep learning to know what questions to ask.
 
-Data simulation is a powerful approach to creating additional data for model testing.
+Data simulation is a powerful approach for understanding your data and how it behaves.
```
Comment (cgreene) [inline content/06.know-your-problem.md:35]:
```suggestion
Data simulation is a powerful approach to develop an understanding of how data and analytical methods interact.
```

File: content/06.know-your-problem.md [hunk 2]
```diff
@@ -32,10 +32,15 @@ For a large dataset, overfitting may not be a concern, but the modeling power of
 It is important to note that molecular or imaging datasets often require appropriate clinical or demographic data to support robust analyses; this must always be balanced with the need to protect patient privacy (see [Tip 10](#privacy)).
 Looking at these annotations can also clarify the study design (for example, by seeing if all the individuals are adolescents or women) or at least help the analyst employing deep learning to know what questions to ask.
 
-Data simulation is a powerful approach to creating additional data for model testing.
+Data simulation is a powerful approach for understanding your data and how it behaves.
 In data simulation, a model is used to learn the true distribution of a training set for the purpose of creating new data points.
-Simulations should be performed under reasonable assumptions since the goal would be to identify useful model architectures and hyperparameters, and simulated datasets can be used to verify the correctness of a model’s implementation.
+Simulations should be performed under reasonable assumptions since the goal would be to identify useful model architectures and hyperparameters.
```
Comment (cgreene) [inline content/06.know-your-problem.md:37]:
```suggestion
Often, researchers may perform simulations under some assumptions about the data generating process to identify useful model architectures and hyperparameters.
```

File: content/06.know-your-problem.md [hunk 3]
```diff
@@ -32,10 +32,15 @@ For a large dataset, overfitting may not be a concern, but the modeling power of
...
 Looking at these annotations can also clarify the study design (for example, by seeing if all the individuals are adolescents or women) or at least help the analyst employing deep learning to know what questions to ask.
 
-Data simulation is a powerful approach to creating additional data for model testing.
+Data simulation is a powerful approach for understanding your data and how it behaves.
 In data simulation, a model is used to learn the true distribution of a training set for the purpose of creating new data points.
-Simulations should be performed under reasonable assumptions since the goal would be to identify useful model architectures and hyperparameters, and simulated datasets can be used to verify the correctness of a model’s implementation.
+Simulations should be performed under reasonable assumptions since the goal would be to identify useful model architectures and hyperparameters.
+Simulated datasets can be used to verify the correctness of a model’s implementation.
 To accurately test the performance of the model, it is important that simulated datasets be generated for a range of parameters.
```
Comment (cgreene) [inline content/06.know-your-problem.md:39]:
```suggestion
To accurately test the performance of the model, it is important that simulated datasets be generated for a range of plausible parameters.
```

File: content/06.know-your-problem.md [hunk 4]
```diff
@@ -32,10 +32,15 @@ For a large dataset, overfitting may not be a concern, but the modeling power of
...
 
-Data simulation is a powerful approach to creating additional data for model testing.
+Data simulation is a powerful approach for understanding your data and how it behaves.
 In data simulation, a model is used to learn the true distribution of a training set for the purpose of creating new data points.
-Simulations should be performed under reasonable assumptions since the goal would be to identify useful model architectures and hyperparameters, and simulated datasets can be used to verify the correctness of a model’s implementation.
+Simulations should be performed under reasonable assumptions since the goal would be to identify useful model architectures and hyperparameters.
+Simulated datasets can be used to verify the correctness of a model’s implementation.
 To accurately test the performance of the model, it is important that simulated datasets be generated for a range of parameters.
 For example, varying the parameters to violate the model's assumptions can test the sensitivity of the model's performance.
+Parameter tuning your simulation model can help you to understand the important features that drive the structure in your dataset.
```
Comment (cgreene) [inline content/06.know-your-problem.md:41]:
```suggestion
Parameter tuning the simulation can help researchers identify the key features that drive method performance.
```

File: content/06.know-your-problem.md [hunk 5]
```diff
@@ -32,10 +32,15 @@ For a large dataset, overfitting may not be a concern, but the modeling power of
...
+Given a simulation model, we can simulate how noise models affect the underlying signal in your dataset.
```
Comment (cgreene) [inline content/06.know-your-problem.md:42]:
```suggestion
In other cases, neural networks can be used to simulate data to better understand how to structure analyses.
```

File: content/06.know-your-problem.md [hunk 6]
```diff
@@ -32,10 +32,15 @@ For a large dataset, overfitting may not be a concern, but the modeling power of
...
+For example some generative neural network models allow you to simulate gene expression data and test the effect of varying number of noise sources on the ability to detect the underlying biological signal in the data [@doi:10.1101/2020.05.03.066597].
```
Comment (cgreene) [inline content/06.know-your-problem.md:43]:
```suggestion
For example, it is possible to study how analytical strategies cope with varying number of noise sources by using neural networks to simulate genome-wide data [@doi:10.1101/2020.05.03.066597].
```

File: content/06.know-your-problem.md [hunk 7]
```diff
@@ -32,10 +32,15 @@ For a large dataset, overfitting may not be a concern, but the modeling power of
...
+Additionally, these generative models can simulate new hypothetical experiments that can be used to answer questions where the data is not available.
```
Comment (cgreene) [inline content/06.know-your-problem.md:44]:
```suggestion
Simulating data from assumptions about the data generating distribution can help to debug or characterize deep learning models, and deep learning models can also simulate data in cases where it is hard to make reasonable assumptions from first principles.
```

**Review by cgreene — APPROVED** [review]
I had a few suggestions to move us away from the second person voice 😂  (though I know we use that elsewhere). I'd say this looks good to me though!

**Comment by Benjamin-Lee** [comment]
Welcome @ajlee21 and thanks for the contributions! I've invited you as a collaborator on the project, which will give you permissions to review and merge others' PRs. Can you add a change in this PR to [`contributors.md`](https://github.com/Benjamin-Lee/deep-rules/blob/master/contributors.md) with your information on a new line?

**Comment by ajlee21** [comment]
Thank you so much for the invitation @Benjamin-Lee! This manuscript looks very interesting and is already teaching me so much! I believe I've added myself as a contributor. And I'll plan on taking a look at some of the other sections this weekend.

**Pull request merged**